src/plugin.py

```diff
@@ -167,7 +168,7 @@ def __init__(self, session, sitemap):
         
         self.refreshTimer = eTimer()
         self.refreshTimer.callback.append(self.refresh_data)
-        self.refreshTimer.start(config_root.refresh.value*1000)
+        self.refreshTimer.start(config_root.refresh.value*100)
```

Comment:
Why do you need resolution greater than seconds? What's your configured value? I don't think this is a good idea, because it pulls the whole sitemap during refresh, so it is going to kill your network. What's your motivation to this change?

Reply:
Retirn to default value 1000ms

Reply:
This would be ideal and most of OH client does update only changed values (through websockets) but this plugin is intended to be simply as much as possible. My use case is to start the plugin, turn off/on some lights in the living room and close the window. I don't thing it makes sense to build "real" OH client for such purpose.